docker/build-project.sh

```diff
@@ -133,14 +137,15 @@ fi
 # Push buildhistory into machine-specific branch in the master buildhistory
 #
 if [ ! -z ${JOB_NAME+x} ]; then
```

Comment:
Yes let's fix that in this PR still.

My point is we don't keep buildhistory settings in refkit-ci.inc anymore. It'd be handled in the same we the archiver mode. The main bitbake build still gets buildhistory if we write the buildhistory settings in this if block.